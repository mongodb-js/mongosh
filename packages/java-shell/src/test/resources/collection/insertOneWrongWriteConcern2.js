// before
db.coll.remove({});
// command
db.coll.insertOne({a: 1}, {writeConcern: {w: -1}});
// command
db.coll.find();
