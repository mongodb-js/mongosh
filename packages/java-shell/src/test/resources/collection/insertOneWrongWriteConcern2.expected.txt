java.lang.IllegalArgumentException: state should be: w >= 0
[ ]